library/std/src/sys/unsupported/common.rs

```diff
@@ -4,8 +4,6 @@ pub mod memchr {
     pub use core::slice::memchr::{memchr, memrchr};
 }
 
-pub use crate::sys_common::os_str_bytes as os_str;
```

Comment:
Should `os_str` be here? Both `wasi` and `wasm` do `use common::*` but define `os_str` themselves anyway.